src/test/java/com/liveramp/cascading_ext/combiner/lib/TestHyperLogLogPlusBackwardsCompatibility.java

```diff
@@ -0,0 +1,29 @@
+package com.liveramp.cascading_ext.combiner.lib;
+
+import com.clearspring.analytics.stream.cardinality.HyperLogLogPlus;
+import org.apache.commons.io.IOUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.*;
+
+public class TestHyperLogLogPlusBackwardsCompatibility {
```

Comment:
quick comment that this is a regression test because we noticed the 2.5 wire format now includes a version number, you could link to the commit we found that shows this

Reply:
new flag introduced in 2.5.0 serialization: https://github.com/addthis/stream-lib/blob/v2.5.0/src/main/java/com/clearspring/analytics/stream/cardinality/HyperLogLogPlus.java#L806

equivalent method in our fork: https://git.liveramp.net/MasterRepos/stream_lib_internal/blob/master/src/main/java/com/clearspring/analytics/stream/cardinality/HyperLogLogPlus.java#L803